lib/screens/forgot_password/cubit/forgot_password_state.dart

```diff
@@ -0,0 +1,20 @@
+part of 'forgot_password_cubit.dart';
+
+@immutable
+abstract class ForgotPasswordState {}
+
+class ForgotPasswordInitial extends ForgotPasswordState {}
```

Comment:
Since this is a multi step screen, the states should also be for each step. Eg:
- `ForgotEmailInitial`, `ForgotEmailError`, `ForgotEmailSuccess`, `ForgotEmailFailure`
- `ForgotOTPInitial`, `ForgotOTPError`, `ForgotOTPSuccess`, `ForgotOTPFailure`
- `ForgotChangeInitial`, ...
- `ForgotLoading` cause loading is same for all steps.

removed **Password** from the names coz they are too long.